test/helpers/runSolidityTest.js

```diff
@@ -37,6 +37,9 @@ const processResult = txReceipt => {
       throw new Error(log.args.message)
     }
   })
+  if (mustAssert && !decodedLogs.length) {
+    throw new Error('No assertions made')
+  }
```

Comment:
Don't know if this is necessary, test fwks usually don't throw if there are no assertions, they simply throw in case there was at least one failure.

Reply:
Yes... it is kind of "special" behaviour but for our purposes I think having this check is more robust:

1. Tests without explicit assertions made are probably buggy in some way
1. This implicitly tests that the test framework is working (decoding `Assert`'s events), avoiding an earlier issue where the tests were always just passing regardless of actual test behaviour.